policyengine_us/variables/gov/states/wv/tax/income/subtractions/senior_citizen_disability_deduction/wv_senior_citizen_disability_deduction.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class wv_senior_citizen_disability_deduction(Variable):
```

Comment:
could we make a `wv_senior_citizen_disability_deduction_person` with all the code up to 
total_reduced_amount = eligible_person * reduced_deduction_amount

and use an `adds` function in the `wv_senior_citizen_disability_deduction`